test/helpers/fixtures.js

```diff
@@ -0,0 +1,22 @@
+import db from '../../src/services/db';
+import * as baseSql from '../../src/base/sql';
+
+const data = {
+  state: require('../fixtures/state')
+};
+
+export default function fixtures () {
+  // Array of promises representing table-insert batch transactions
+  const insertPromises = [];
+  for (const table in data) {
+    // Insert data for each table
+    insertPromises.push(db.tx(t => {
+      return t.batch(data[table].map(insert => {
+        const fields = Object.keys(insert);
```

Comment:
Huh. This one's going to take some unraveling. Right now, `src/services/db` has a single (default) export that is the resulting `db` object from munged config options. That is, I don't have access to the `pgp` library itself from that exported object (I don't think? `pg-promise` is new to me and I haven't gotten my sea legs yet). Here are the options that leap to mind:

* Add an additional, named, export to `db`. Pros: easy as pie. Cons: combining named and default exports. I don't know what our position is on that :).
* Export something else (e.g. the entire `pgp` object) from `db`. Yikes.
* Re-initialize `pgp` in places where it's needed. Also yikes (let's leave config munging in one place, eh?).

I guess given those options I'll probably head toward the multiple-exports-from-`db` one. I think we'll probably run into other places we need access to the `pgp` library itself...you think? Anyway, I'll check with `#code-review` to see if there is a backlash against this form of exporting and go that way if not.